usr/local/www/diag_system_activity.php

```diff
@@ -82,31 +81,45 @@ function activitycallback(transport) {
 	if($savemsg) {
 		echo "<div id=\"savemsg\">";
 		print_info_box($savemsg);
-		echo "</div>";	
+		echo "</div>";
 	}
 	if ($input_errors)
 		print_input_errors($input_errors);
 ?>
-<table width="100%" border="0" cellpadding="0" cellspacing="0" summary="diag system activity">
-  <tr>
-    <td>
-	<table id="backuptable" class="tabcont" align="center" width="100%" border="0" cellpadding="6" cellspacing="0" summary="tabcont">
-		<tr>
-			<td align="center">
-				<table summary="results">
-					<tr><td>
-						<div id="cpuactivitydiv">
-							<?=gettext("Gathering CPU activity, please wait...");?>
-						</div>
-					</td></tr>
-				</table>
-			</td>
-		</tr>
-	</table>
-    </td>
-  </tr>
-</table>
+
+    <div class="table-responsive">
+        <table class="table table-striped table-hover" summary="diag system activity">
```

Comment:
I agree. It can be hard to find the right compromise between only touching the pieces affected by bootstrap and fixing the original php with may no longer be optimal. I’ll “un-nest" the tables this time :)

Steve

> On Apr 7, 2015, at 9:25 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_system_activity.php <https://github.com/SjonHortensius/pfsense/pull/62#discussion_r27876793>:
> 
> > -				<table summary="results">
> > -					<tr><td>
> > -						<div id="cpuactivitydiv">
> > -							<?=gettext("Gathering CPU activity, please wait...");?>
> > -						</div>
> > -					</td></tr>
> > -				</table>
> > -			</td>
> > -		</tr>
> > -	</table>
> > -    </td>
> > -  </tr>
> > -</table>
> > +
> > +    <div class="table-responsive">
> > +        <table class="table table-striped table-hover" summary="diag system activity">
> Summary-attribute is deprecated and should be removed
> I dislike the nested tables and this is yet another example where it is unwanted. In this case, why do you even need 3 wrapping tables if the content is a DIV ?
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/62/files#r27876793>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>